Assets/PlayerAssets/Scripts/Stats/CharacterStats.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CharacterStats : MonoBehaviour {
+
+ 		public int maxHealth = 100;
+        public int currentHealth { get; private set;}
+
+        public Stat damage;
```

Comment:
set defaults to these, doesn't matter if you overwrite them in the inspector for now. We don't want our code exploding bc there is no value when we first attach this script to an object. We should always avoid saving final values of things in the inspector, use it as a tool to find a good value, then set that value in code. 
This will eliminate issues with prefabs and our scenes functioning very different based on arbitrary inspector values.

In the future we are going to be generating players, enemies and everything by code using values in the code, and we won't use the inspector at all for anything but dev